stackslib/src/chainstate/nakamoto/mod.rs

```diff
@@ -1696,7 +1784,8 @@ impl NakamotoChainState {
         db_handle: &mut SortitionHandleConn,
         staging_db_tx: &NakamotoStagingBlocksTx,
         headers_conn: &Connection,
-        aggregate_public_key: &Point,
+        _aggregate_public_key: &Point,
```

Comment:
You can remove this argument, and just note in the comment below that it will be required in the future.

Reply:
I made this an `Option` (still unused), as I'm not sure whether this should or shouldn't be a place where we want to refactor later (given how it's used downstream).